packages/runtime-core/src/componentProps.ts

```diff
@@ -140,7 +140,7 @@ export function updateProps(
   } = instance
   const rawOptions = instance.type.props
   const rawCurrentProps = toRaw(props)
-  const { 0: options } = normalizePropsOptions(rawOptions)
+  const [ options ] = normalizePropsOptions(rawOptions)
```

Comment:
The prettier setup doesn't add spaces around array destructuring brackets.
```suggestion
  const [options] = normalizePropsOptions(rawOptions)
```